hardhat.config.ts

```diff
@@ -91,12 +91,31 @@ const config: HardhatUserConfig = {
   etherscan: {
     apiKey: {
       mainnet: env.string("ETHERSCAN_API_KEY_ETH", ""),
-      goerli: env.string("ETHERSCAN_API_KEY_ETH", ""),
-      arbitrumGoerli: env.string("ETHERSCAN_API_KEY_ARB", ""),
+      sepolia: env.string("ETHERSCAN_API_KEY_ETH", ""),
+      // arbitrumSepolia: env.string("ETHERSCAN_API_KEY_ARB", ""),
       arbitrumOne: env.string("ETHERSCAN_API_KEY_ARB", ""),
       optimisticEthereum: env.string("ETHERSCAN_API_KEY_OPT", ""),
-      optimisticGoerli: env.string("ETHERSCAN_API_KEY_OPT", ""),
+      "opt_sepolia": env.string("ETHERSCAN_API_KEY_OPT", ""),
```

Comment:
The repo uses "prettier" for code formatting, but currently, it's not configured to work with pre-commit hook (for example, https://github.com/lint-staged/lint-staged?tab=readme-ov-file). Consider adding pre-commit linting for the files (at least typescript files) and enabling auto-formatting in the IDE.